### ISSUE TITLE
Incompatible with Angular 6.0

### ISSUE THREAD

**Issue by yoginm** [mariuszfoltak/angular2-datatable#156]
Get following error after upgrade to Angular 6.0

node_modules/angular2-toaster/src/toaster.service.d.ts(3,10): error TS2305: Module '"Seller/node_modules/rxjs/Observable"' has no exported member 'Observable'.
node_modules/angular2-toaster/src/toaster.service.d.ts(5,10): error TS2305: Module '"Seller/node_modules/rxjs/Subject"' has no exported member 'Subject'.

**Comment by chrisbmoore** [comment]
It has to do with the Rxjs 6 changes. Considering the project hasn't even been updated to support Angular 4, I doubt Angular 6 support will get added soon.

Workaround fix could be to install rxjs-compat package:
> npm install --save rxjs-compat

**Comment by yoginm** [comment]
Yes, I see that we can work temporarily with rxjs-compat. I just want to see if this will be updated. rxjs-compat adds overhead to my application. At some point it will need to be taken out. Hoping this would be updated to rxjs 6..

**Comment by PavolSvidran** [comment]
Hi, if it is possible to upgrade it, it's a simple change, if you want help, I will help you.
RxJs-compat is redundant package I want to get rid of as soon as it goes, as @yoginm note it overhead application and after RxJs7 it will not work anymore.

**Comment by PascalHonegger** [comment]
I created a fork (of a fork...) which has the minimal changes required for angular 6 to run (https://github.com/PascalHonegger/ng-datatable - https://www.npmjs.com/package/@pascalhonegger/ng-datatable). We use this version for our project and were able to get rid of the rxjs-compat package. I'm also happy to keep it updated / merged wherever an active version of this package lives.
